dcm4che-net-audit/pom.xml

```diff
@@ -8,6 +8,9 @@
   <artifactId>dcm4che-net-audit</artifactId>
   <name>dcm4che-net-audit</name>
   <packaging>bundle</packaging>
+  <properties>
+    <powermock.version>2.0.9</powermock.version>
```

Comment:
I don't think adding a property makes much sense if this is not a group of projects and it is only used once?

Reply:
Not big deal. Initially I tried to add more unit tests and had more depedencies. If I have chance to update, either way will work.